cmd/build/helmify/kustomize-for-helm.yaml

```diff
@@ -102,3 +102,41 @@ metadata:
   annotations: HELMSUBST_SECRET_ANNOTATIONS
   name: gatekeeper-webhook-server-cert
   namespace: gatekeeper-system
+---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  labels:
+    gatekeeper.sh/system: "yes"
+  name: gatekeeper-validating-webhook-configuration
+webhooks:
+- clientConfig:
+    service:
+      name: gatekeeper-webhook-service
+      namespace: gatekeeper-system
+      path: /v1/admit
+  name: validation.gatekeeper.sh
+  timeoutSeconds: HELMSUBST_VALIDATING_WEBHOOK_TIMEOUT
+  rules:
+  - apiGroups:
+    - "*"
+    apiVersions:
+    - '*'
+    operations: HELMSUBST_VALIDATING_WEBHOOK_OPERATION_RULES
+    resources:
+    - '*'
+- clientConfig:
+    service:
+      name: gatekeeper-webhook-service
+      namespace: gatekeeper-system
+      path: /v1/admitlabel
+  name: check-ignore-label.gatekeeper.sh
+  timeoutSeconds: HELMSUBST_VALIDATING_WEBHOOK_TIMEOUT
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - '*'
+    operations: HELMSUBST_VALIDATING_WEBHOOK_OPERATION_RULES
```

Comment:
I don't think DELETE is relevant for the check-ignore-label webhook?

We ignore it in the code, so probably best if we just never call it for DELETE to save on QPS and avoid service interruptions:

https://github.com/open-policy-agent/gatekeeper/blob/6243e45a73f10bd67f0890c42e6f2cdb6ae6ce20/pkg/webhook/namespacelabel.go#L71-L73

Reply:
removed it